build.gradle.kts

```diff
@@ -0,0 +1,199 @@
+import org.gradle.api.tasks.bundling.Jar
+import kotlin.concurrent.thread
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.io.InputStream
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+
+plugins {
+    `java-library`
+    `maven-publish`
+    kotlin("jvm") version "1.2.61"
+    id("net.saliman.properties") version "1.4.6"
+}
+
+fun gitBranch() =
+        if (System.getenv("TRAVIS_BRANCH") != null) System.getenv("TRAVIS_BRANCH")!!
+        else {
+            val r = "git rev-parse --abbrev-ref HEAD".execute(rootDir.absolutePath)
+            r.lineSequence().last()
+        }
+
+val branch = gitBranch()
```

Comment:
Why declare a separate function? These 8 lines could be simplified to this single line:

```kotlin
val branch = System.getenv("TRAVIS_BRANCH") ?: "git rev-parse --abbrev-ref HEAD".execute(rootDir.absolutePath).lines().last()
```

Reply:
Yeah I just copied it over from the older groovy version, I'll fix it :')